index.js

```diff
@@ -31,15 +27,12 @@ module.exports = {
       },
       label: 'Please choose a year',
       placeholder: 'year',
-      options: {
-        minYear: '1900',
-        maxYear: '2016',
-      },
       type: 'select',
       settings: {
-        empty: true,
+        minYear: 2000,
+        maxYear: 2016,
       },
     },
   },
-  html: '<label for="{{year.id}}">{{year.label}}</label><select id="{{year.id}}" name="{{year.name}}"><option value="{{year.value}}"  class="year-range"></option></select></label>',
+  html: '<label for="{{year.id}}">{{year.label}}</label><select id="{{year.id}}" name="{{year.name}}">{% for i in range(year.settings.minYear, year.settings.maxYear) -%}<option value="{{ i }}">{{ i }}</option>{%- endfor %}</select></label>',
```

Comment:
Need check to see if `year.value` is equal to `i` to set the `selected` attribute on the `option` tag